src/main/scala/cromwell/binding/values/WdlFile.scala

```diff
@@ -11,7 +12,7 @@ object WdlFile {
   }
 }
 
-case class WdlFile(value: String) extends WdlPrimitive {
+case class WdlFile(value: String, globbiness: GlobOrNot = GlobOrNot.NotGlob) extends WdlPrimitive {
```

Comment:
Seeing this here makes me desire the "not a field" angle even most strongly now :)